src/main/java/com/seller/panel/controller/RegistrationController.java

```diff
@@ -0,0 +1,42 @@
+package com.seller.panel.controller;
+
+import com.seller.panel.handler.ExceptionHandler;
+import com.seller.panel.service.MailService;
+import com.seller.panel.util.EndPointConstants;
+import com.seller.panel.util.JwtTokenUtil;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.constraints.NotNull;
+
+@RestController
+@RequestMapping
+public class RegistrationController {
+
+    @Autowired
+    private ExceptionHandler exceptionHandler;
```

Comment:
Create a BaseController for exception handling and extends all controller from it or use composition, remove all boilerplate logic for getting exception there.